src/ComponentFolders/LoginPage/Login.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import Image from "./Images/LoginSofa.svg"
+
```

Comment:
![Screenshot_20221227_122513](https://user-images.githubusercontent.com/110013797/209644567-e51a9978-3f0a-42f3-891a-43c09975c19c.png)

Facebook and Google logins have not been added. The images are available in the images folder.

Reply:
Okay, I will add them immediately